src/Api/ItemList.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+namespace OpsWay\ZohoBooks\Api;
+
+
+class ItemList implements \ArrayAccess, \IteratorAggregate
+{
+    /**
+     * @var iterable
+     */
+    private $items = [];
+
+    /**
+     * @var iterable
+     */
+    private $pageContext = [];
+
+    function __construct(iterable $items,iterable $pageContext)
```

Comment:
Iterable type here have no sense, we always parse response to array, so array type more preferred. BTW, library should be compatible with PHP 5.6, off course we can drop 5.6 support (soon it will legacy - http://php.net/supported-versions.php), but iterable type appear only in 7.1.